src/js/utils/mercator.js

```diff
@@ -829,14 +829,22 @@ mercator.getCircleStyle = (radius, fillColor, border) =>
   new Style({
     image: new CircleStyle({
       radius,
-      fill: new Fill({ color: fillColor || "rgba(255, 255, 255, 0)" }),
+      fill: new Fill({ color: fillColor || "tansparent" }),
```

Comment:
can we revert this change, as what we need to do is to inject the style directly in the xml?